proto/settings.proto

```diff
@@ -12,5 +12,6 @@ message SettingsMsg {
   int32 wifi_power = 8;
   bool is_locked = 9; // if the board is actively locked/parked
   bool locking_enabled = 10; // if the board locking functionality is enabled ('armed')
-  reserved 11;
+  bool monitoring_only_enabled = 11;
+  reserved 12;
```

Comment:
Reserved IDs are for the fields that have been removed. These IDs shouldn't be reused, that's why it's called reserved. I'd also call the boolean `monitoring_only_mode`.

So this should be:
```
bool monitoring_only_mode = 12;
reserved 11;
```